static/swagger-specs/reactor.yaml

```diff
@@ -11138,6 +11361,533 @@ components:
+                          properties:
+                            id:
+                              type: string
+                              description: "The [delegate descriptor ID](https://experienceleague.adobe.com/docs/experience-platform/tags/api/guides/delegate-descriptor-ids.html) for the configuration."
+                              example: "EP43649cc8856d4f09a7c2a21a4b1e449d"
+                            type:
+                              type: string
+                              description: The type of resource being created.
+                              example: "extension_packages"
+                links:
+                  type: object
+                  properties:
+                    self:
+                      type: string
+                      description: A link that can be used in a subsequent API call to fetch the relationship object for the resource's related data elements.
+                      example: "https://reactor.adobe.io/extension_package_usage_authorizations/EA35d0e731f73645e6972df9fcac101434"
+        meta:
+          type: object
+          properties:
+            pagination:
+              type: object
+              properties:
+                current_page:
+                  type: integer
+                  description: The current page of the response.
+                  example: 1
+                next_page:
+                  type: integer
+                  description: The next page of the response.
+                  example: "null"
+                prev_page:
+                  type: integer
+                  description: The previous page of the response.
+                  example: "null"
+                total_pages:
+                  type: integer
+                  description: The total number of pages in the response.
+                  example: 1
+                total_count:
+                  type: integer
+                  description: The total number of results in the response.
+                  example: 1
+              description: Contains pagination information about the list response.
+          description: Contains a `pagination` object that provides pagination information about the list response.
+    extensionPackagesUpdateUsageAuthorizationPayload:
+      type: object
+      properties:
+        data:
+          type: object
+          properties:
+            attributes:
+              type: object
+              properties:
+                state:
+                  type: string
+                  description: The current state of the updated extension package user authorization.
+                  example: approved
+            type:
+              type: string
+              description: The type of resource being created.
+              example: "extension_package_usage_authorizations"
+            id:
+              type: string
+              description: The authorized Organization ID.
+              example: "EA86f54b48dd7042a68686508e03be8ba9"
```

Comment:
I'm not sure if these two match up with each other? Since the example doesn't look like an org ID.